modules/space/rendering/renderablefluxnodes.cpp

```diff
@@ -0,0 +1,1093 @@
+        "impacting the node size."
+    };
+    constexpr openspace::properties::Property::PropertyInfo MinNodeSizeInfo = {
+        "minNodeSize",
+        "Minimum node size",
+        "The minimum node size."
+    };
+    constexpr openspace::properties::Property::PropertyInfo MaxNodeSizeInfo = {
+        "maxNodeSize",
+        "Maximum node size",
+        "The minimum node size."
+    };
+    constexpr openspace::properties::Property::PropertyInfo AlwaysPulseInfo = {
+        "alwaysPulsate",
+        "Pulsate regardless of camera position",
+        "Always have nodes close to earth pulsate regardless of position."
+    };
+    constexpr openspace::properties::Property::PropertyInfo pulseEnabledInfo = {
+       "pulseEnabled",
+       "Nodes close to Earth pulsate",
+       "Toggles the pulse for nodes close to Earth."
+    };
+    constexpr openspace::properties::Property::PropertyInfo gaussianPulseEnabledInfo = {
+       "gaussianPulseEnabled",
+       "Nodes close to Earth pulsate with alpha by gaussian",
+       "Toggles the pulse with alpha by gaussian for nodes close to Earth."
+    };
+
+    float stringToFloat(const std::string input, const float backupValue = 0.f) {
+        float tmp;
+        try {
+            tmp = std::stof(input);
+        }
+        catch (const std::invalid_argument& ia) {
+            LWARNING(fmt::format(
+                "Invalid argument: {}. '{}' is NOT a valid number", ia.what(), input
+                ));
+            return backupValue;
+        }
+        return tmp;
+    }
+
+    struct [[codegen::Dictionary(RenderableFluxNodes)]] Parameters {
+        // path to source folder with the 3 binary files in it
+        std::string sourceFolder;
+        //
+        struct TransferFunctions {
+            std::string standard;
+            std::string flow;
+            std::string earth;
+            std::string cmr [[codegen::key("CMR")]];
+        };
+        // [[codegen::verbatim(ColorTablePathInfo.description)]]
+        TransferFunctions colorTablePaths;
+        // [[codegen::verbatim(LineWidthInfo.description)]]
+        //float lineWidth;
+        // [[codegen::verbatim(GoesEnergyBinsInfo.description)]]
+        std::optional<int> energyBin;
+    };
+#include "renderablefluxnodes_codegen.cpp"
+
+    // Changed everything from dvec3 to vec3
+    glm::vec3 sphericalToCartesianCoord(glm::vec3 position) {
+        glm::vec3 cartesianPosition = glm::vec3();
+
+        // ρsinφcosθ 
+        cartesianPosition.x = position.x * sin(position.z) * cos(position.y);
+        // ρsinφsinθ
+        cartesianPosition.y = position.x * sin(position.z) * sin(position.y);
+        // ρcosφ
+        cartesianPosition.z = position.x * cos(position.z);
+
+        return cartesianPosition;
+    }
+} //namespace
+
+namespace openspace {
+
+documentation::Documentation RenderableFluxNodes::Documentation() {
+    return codegen::doc<Parameters>("space_renderable_flux_nodes");
+}
+
+using namespace properties;
```

Comment:
Not a big a fan of such broad `using namespace` for the whole file as it makes things harder to read